packages/components/src/components/Log/Log.js

```diff
@@ -82,6 +82,11 @@ export class LogContainer extends Component {
           id: 'dashboard.pipelineRun.stepFailed',
           defaultMessage: 'Step failed'
         });
+      case 'Not run':
```

Comment:
no need to display a trailer on the logs for this case

Reply:
True, it's not really necessary. I'll remove it 👍